packages/core/src/lib/core/initialise-lists.ts

```diff
@@ -48,17 +48,30 @@ export type InitialisedField = Omit<NextFieldType, 'dbField' | 'access' | 'graph
     cacheHint: CacheHint | undefined;
   };
   ui: {
+    label: string; // TODO: move to ui
+    description: string;
+    views: string;
     createView: {
       fieldMode: MaybeSessionFunction<'edit' | 'hidden', any>;
     };
     itemView: {
       fieldMode: MaybeItemFunction<'read' | 'edit' | 'hidden', any>;
+      fieldPosition: MaybeItemFunction<'form' | 'sidebar', any>;
     };
     listView: {
       fieldMode: MaybeSessionFunction<'read' | 'hidden', any>;
     };
   };
-};
+} & Pick<
+  NextFieldType,
+  | 'input'
+  | 'output'
+  | 'getAdminMeta'
+  | 'views'
+  | '__ksTelemetryFieldTypeName'
+  | 'extraOutputFields'
+  | 'unreferencedConcreteInterfaceImplementations'
```

Comment:
This will minimize the number of breaking changes moving forward and keeps us keenly aware of what else is being copied to this type.